Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -511,11 +507,12 @@ public class TestConductRepository : ITestConductRepository
                             }
 
                             testAnswers.TestConduct = testConduct;
-                            await _dbContext.TestAnswers.AddAsync(testAnswers);
-                            await _dbContext.SaveChangesAsync();
+                            testAnswersList.Add(testAnswers);
                         }
-                        
                     }
+                    await _dbContext.TestAnswers.AddRangeAsync(testAnswersList);
```

Comment:
Why are you adding this two times?

Reply:
one TestAnswer table and other one is TestConduct table

Reply:
:smile: sunday mornings

Reply:
Rebase it